code/controllers/subsystems/initialization/materials.dm

```diff
@@ -40,6 +40,8 @@ SUBSYSTEM_DEF(materials)
 	. = ..()
 
 /datum/controller/subsystem/materials/proc/get_material_by_name(name)
+	if(!name || name == "")
```

Comment:
Merely a note:
`!name` includes `name == "" ` due to how BYOND handles falsy values.

Reply:
As discussed on Discord, I'm too lazy to fix this just now. Will tweak it in a future PR.

Reply:
_I lied._